Getting Started Guide (#96)
NCAR/geocat-applications: README.md

in NCL in Python, including any algorithmic differences, guidance regarding replication under different

> conditions or circumstances, and any other relevant comparisons between the NCL and Python functionality

Reviewer: ```suggestion
conditions or circumstances, and any other relevant comparisons between the NCL and Python functionality.
```

Reviewer: Actually this whole page is missing periods at the end of paragraphs. Is that a stylistic choice?

Author: It was, but I can add them 👍

> This is part of the GeoCAT's commitment to NCL and NSF NCAR's [Pivot to Python Initiative](https://www.ncl.ucar.edu/Document/Pivot_to_Python/)

Reviewer: ```suggestion
This is part of the GeoCAT's commitment to NCL and NSF NCAR's [Pivot to Python Initiative](https://www.ncl.ucar.edu/Document/Pivot_to_Python/).
```

> This is part of the GeoCAT's commitment to NCL and NSF NCAR's [Pivot to Python Initiative](https://www.ncl.ucar.edu/Document/Pivot_to_Python/)

Author: New issue: https://github.com/NCAR/geocat-applications/issues/98

in NCL in Python, including any algorithmic differences, guidance regarding replication under different
conditions or circumstances, and any other relevant comparisons between the NCL and Python functionality

> This is part of the GeoCAT's commitment to NCL and NSF NCAR's [Pivot to Python Initiative](https://www.ncl.ucar.edu/Document/Pivot_to_Python/)

Reviewer 2: ```suggestion
This is part of the GeoCAT team's commitment to NCL and NSF NCAR's [Pivot to Python](https://www.ncl.ucar.edu/Document/Pivot_to_Python/)
```
I think this is a little more clear / correct. 

It'd be good to get @erogluorhan's eyes on this as well to make sure this is worded in a way he's comfortable with (re: the GeoCAT team commitment).

Reviewer 3: This reworded one sounds good to me even though I'd find the previous one also fine. Thanks!
